src/torch_models.py

```diff
@@ -453,15 +453,20 @@ def fit(self, X: Array, y: Array) -> None:
             member.fit(X, y)
 
     def classify(self, x: Array) -> bool:
-        member_vals = []
-        for member in self._members:
-            x_normalized = member.normalize(x)
-            member_vals.append(member(x_normalized).item())
-        avg = np.mean(member_vals)
+        avg = np.mean(self.logits(x))
         classification = avg > 0.5
         assert classification in [True, False]
         return classification
 
+    def logits(self, x: Array) -> Array:
+        """Return logits calculated by each member."""
+        assert x.ndim == 1
+        logits = []
+        for member in self._members:
+            x_normalized = member.normalize(x)
+            logits.append(member(x_normalized).item())
```

Comment:
I'm getting turned around about whether the output of `member(*)` is a logit or just a probability. I think I stated the definition of logit wrong when we were talking about it earlier. A logit should be `log(p / (1- p))`, right? But I don't think that's what we want it to be here, nor do I think that's what it actually is -- I think this is really just the probability for class True. And for the calculation of entropy and the BALD objective, that's what we want it to be, too. If you agree, can we rename this function to `predict_proba`, in line with sklearn? Sorry for my confusion!

Reply:
oh yes, that's my bad! i forgot that's the actual definition of logit. i will rename the function